pennylane/transforms/qfunc_transforms.py

```diff
@@ -68,7 +68,7 @@ def wrapper(*args, **kwargs):
         active_tape = qml.tape.get_active_tape()
 
         if active_tape is not None:
-            with active_tape.stop_recording(), active_tape.__class__() as tape:
+            with active_tape.stop_recording(), active_tape.__class__() as tape:  # pylint: disable=not-context-manager

```

Comment:
Hmm, should the context manager block be changed here? :thinking: How come the linting warning started to show?

Reply:
I have no idea why this linting warning appeared, hence why I just silenced it. Problem might be fixed in #3068  ?